### PR TITLE
Python modules virtual environments support

### PR DESCRIPTION
**Pull request by p1c2u**
Run autoflake/black/flake8/isort/mypy with poetry/pipenv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +39,7 @@ export BUILD_HARNESS_EXTENSIONS_PATH ?= $(BUILD_HARNESS_PATH)/build-harness-exte
   kubeval/validate                    Validate manifests
   opa/clone-policy                    Git clone policies (requires OPA_POLICY_REPO argument)
   opa/conftest                        Validate manifests
+  pipenv/lock                         Lock Pipenv dependencies
```
Comment (LiamCato) [inline README.md:42]: Do you want to add a poetry lock command?

Comment (p1c2u) [reply README.md:42]: it is already there

Comment (LiamCato) [reply README.md:42]: Even better!

**Review by LiamCato — APPROVED** [review]

**Review by LiamCato — APPROVED** [review]

**Comment by p1c2u** [comment]
@nickmintel we've updated with @LiamCato  the branch. Can you take a look at it again and merge if everything is okay?

**Pull request merged**